engine/runtime-with-polyglot/src/test/scala/org/enso/interpreter/test/instrument/RuntimeVisualizationsTest.scala

```diff
@@ -27,20 +25,14 @@ class RuntimeVisualizationsTest
     with Matchers
```

Comment:
Can we also have a `.java` test ;-? Only then `.enso-sources*` files are generated and IGV can open the project.